src/main/java/uk/gov/companieshouse/web/pps/controller/pps/ViewPenaltiesController.java

```diff
@@ -84,15 +87,17 @@ public String getViewPenalties(@PathVariable String companyNumber,
             return ERROR_VIEW;
         }
 
-        model.addAttribute("outstanding", lateFilingPenalty.getOutstanding());
-        model.addAttribute("madeUpDate",
-                LocalDate.parse(lateFilingPenalty.getMadeUpDate(),
-                        DateTimeFormatter.ofPattern("uuuu-MM-dd", Locale.UK))
-                        .format(DateTimeFormatter.ofPattern("d MMMM uuuu", Locale.UK)));
+        DecimalFormat formatter = new DecimalFormat("#,###");
+        String formattedOutstanding = formatter.format(lateFilingPenalty.getOutstanding());
+
+        model.addAttribute("referenceTitle", penaltyNumber.startsWith("A") ? "Reference Number" : "Penalty Reference");
+        model.addAttribute("outstanding", formattedOutstanding);
         model.addAttribute("dueDate",
```

Comment:
The latest prototype and comment from Hamayun suggests the `dueDate` row can be removed.
See https://sanctions-payment-prototype-0d6d68d6c82d.herokuapp.com/v9/05-penalty-summary-lfp
and https://companieshouse.atlassian.net/browse/SAN-125?focusedCommentId=269282

I admit that it's quite confusing though!